test/system/320-system-df.bats

```diff
@@ -144,11 +144,11 @@ Size           |   ~${size}.*MB |        !0B |            0B
     run_podman system df --format '{{.Reclaimable}}'
     is "${lines[0]}" ".* (100%)" "100 percent of image data is reclaimable because $IMAGE has unique size of 0"
 
-    # Make sure the unique size is now really 0.  We cannot use --format for
-    # that unfortunately but we can exploit the fact that $IMAGE is used by
-    # two containers.
+    # Note unique size is basically never 0, that is because we count certain image metadata that is always added.
+    # The unique size is not 100% stable either as the generated metadata seems to differ a few bytes each run.
     run_podman system df -v
-    is "$output" ".*0B\\s\\+2.*"
+    is "$output" ".*1[0-9].[0-9]\\+MB.*3.[0-9]\\+kB\\s\\+2.*" "Shared and Unique Size"
+    is "$output" ".*1[0-9].[0-9]\\+MB.*6.[0-9]\\+kB\\s\\+0.*" "Shared and Unique Size"
```

Comment:
What do these two regexes check? Wouldn't it be better to use `assert` instead of `is`?

Reply:
`assert` or `is` should be the same here, because the test `is` using is overall I have opted to keep using is for consistency.

The sizes are non stable (metadat, different storage drivers, etc...) so I have opted to use a regex range that makes sense.

Reply:
```suggestion
    #   regex match:       SHARED SIZE      |         UNIQUE SIZE       |   CONTAINERS
    is "$output" ".*[1-9]\\d*\\.[0-9]\\+MB\\s\\+[1-9]\\d*\\.[0-9]\\+kB\\s\\+2.*" "Shared and Unique Size"
    is "$output" ".*[1-9]\\d*\\.[0-9]\\+MB\\s\\+[1-9]\\d*\\.[0-9]\\+kB\\s\\+0.*" "Shared and Unique Size"      
```
I would use this regex, which is preserved even when updating and resizing the image. 

> Maybe only one line could be checked. Because there is only a difference between the container numbers.

Reply:
Yes that is an option if we want to allow all numbers, it certainly should make it more future proof on image changes.

Reply:
I think that would be better.

Reply:
ok used assert and single quotes which hopefully should make the regex more readable, also I did not add 1-9 extra check as this seems like more complications that don't add any test value.